### ISSUE TITLE
Getting assertion failed with I believe the correct node versions

### ISSUE THREAD

**Issue by JohnAllen** [zodern/meteor-up#441]
I have tried to solve this by reading some of the old mup issues but haven't been able to.  This was on a clean Ubuntu 16 machine.

`dockerImage: 'abernix:meteord:base'`

`meteor node --version v4.6.1` (using meteor 1.4.1.3)

```
WARN cannot run in wd meteor-dev-bundle@0.0.0 node npm-rebuild.js (wd=/bundle/bundle/programs/server)
[]=> Starting meteor app on port:80
[]
[]assert.js:93
[]  throw new assert.AssertionError({
[]        ^
[]AssertionError: "undefined" === "function"
[]    at wrapPathFunction (/bundle/bundle/programs/server/mini-files.js:77:10)
```

Have any idea if this is a node version issue or something else?

**Comment by zodern** [comment]
How did you fix it?

**Comment by JohnAllen** [comment]
Either one or both of two things: 

1) At first I didn't think it was another Node versioning issue, like all of the historically similar issues I found, because both `Meteor node --version` and the Meteor version used by the image were both 4 something (at least that's what others have said the image version was).  And I thought that these were the only two that mattered.  But then just `node --version` locally was running a 5 version and I started to use 4 there, AND/OR
2) I changed `'abernix:meteord:base'` to `'abernix/meteord:base'` in `mup.js` because it said that that wasn't a valid docker image or repo or somethin.  Perhaps this change should be made in the repo?  But then I wonder how it's working for others.  (I have been traveling and half working and sun-pounded so I really don't know what helped, all I know is I did these two things and my last `mup deploy` was successful with using a forward slash there.)
